test/ModuleInterface/SomeDistributedActor.swiftinterface

```diff
@@ -0,0 +1,16 @@
+// swift-interface-format-version: 1.0
+// swift-module-flags: -module-name SomeDistributedActor
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t/DistributedImport.swiftmodule -module-name DistributedImport %S/Inputs/someDistributedActorModule.swift
+// RUN: %target-swift-frontend -compile-module-from-interface -o %t/SomeDistributedActor.swiftmodule %s -I %t -verify
```

Comment:
Regardless of `source null` issue I think it's good to have a round-trip test like we have for property wrappers - https://github.com/apple/swift/blob/main/test/ModuleInterface/property_wrappers.swift. It emits swiftinterface, type-checks it and uses it to emit a module.